reascripts/ReaSpeech/source/ReaSpeechWorker.lua

```diff
@@ -198,9 +185,24 @@ function ReaSpeechWorker:handle_job_status(active_job, response)
   end
 
   if response.job_status == 'SUCCESS' then
-    local transcript_url_path = response.job_result.url_path
+    local job_result = response.job_result
+    if job_result.result then
+      self:handle_response(active_job, job_result.result)
+      self.active_job = nil
+      return false
+    end
```

Comment:
if we have a result, we don't need to worry about followup requests and can stop right here.